### ISSUE TITLE
Update to latest Choices.js version

### ISSUE THREAD

**Issue by froschdesign** [zendframework/zf-mkdoc-theme#69]
https://github.com/jshjohnson/Choices/releases

**Comment by froschdesign** [comment]
@thexpand 
Can you look at this? Your extension with the sticky option groups will no longer work with the latest version of Choices.js.

Thank you in advance!

**Comment by thexpand** [comment]
@froschdesign 
Of course I can. Why exactly it will not work? I tried to figure out which of the releases would eventually break the sticky option groups, but I can't figure out without testing. It would be great to hear what's your idea in particular.

**Comment by froschdesign** [comment]
@thexpand 
I found the problem:

https://github.com/zendframework/zf-mkdoc-theme/blob/234b071c533701777142602dda92e7f9280ff66c/asset/js/component-list.js#L101

must be changed to:

```js
const groups = choices.choiceList.element.querySelectorAll('.choices__group');
```

Choices.js used now a `List` object:

```js
this.choiceList = new List({
    element: this._getTemplate('choiceList', this._isSelectOneElement),
});
```

* https://github.com/jshjohnson/Choices/blob/826384b9d5a3e7fc1575fb0a0545f294705b8dd9/src/scripts/choices.js#L1807-L1809
* https://github.com/jshjohnson/Choices/blob/master/src/scripts/components/list.js

---

**Thanks for your quick response!**

**Comment by thexpand** [comment]
@froschdesign Glad you found it. What I meant is whether you know exactly what's wrong or if I need to do a research, but you've already done it.
Do you want me to test it and submit a PR or anything else to help with?

**Comment by froschdesign** [comment]
Fixed with #5f7be16ac9e17e82cb932e8cda05e241d0379b6d
